packages/fxa-settings/src/components/Settings/PageRecentActivity/en.ftl

```diff
@@ -9,3 +9,5 @@ recent-activity-account-login = Account initiated login
 recent-activity-account-reset = Account initiated password reset
 recent-activity-emails-clearBounces = Account cleared email bounces
 
+## $date (Date) - Date recent activity was created
+recent-activity-created-at = from here { $date }
```

Comment:
What is the "from here"? I don't see this in the fallback text.

Reply:
Ah whoops. This was for testing. I must have forgotten to remove it. Thank you!